src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,34 @@
-export const Tabs = () => {};
+export const Tabs = (
+  { tabs,
+    selectedTabId,
+    onTabSelected },
+) => (
+  <div data-cy="TabsComponent">
+    <div className="tabs is-boxed">
+      <ul>
+        {tabs.map(el => (
+          <li
+            data-cy="Tab"
+            className={(selectedTabId === el.id) ? 'is-active' : ''}
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally